server/src/main/java/org/opensearch/node/resource/tracker/NodeResourceUsageTracker.java

```diff
@@ -69,6 +70,9 @@ public IoUsageStats getIoUsageStats() {
      * Checks if all of the resource usage trackers are ready
      */
     public boolean isReady() {
+        if (Constants.LINUX) {
+            return memoryUsageTracker.isReady() && cpuUsageTracker.isReady() && ioUsageTracker.isReady();
```

Comment:
Dont we need to initialize() for this ioUsageTracker ?

Reply:
Yes it was initialised in `initialize` method.